ui/src/services/utils.js

```diff
@@ -98,3 +98,11 @@ export const useRefreshEffect = (refreshAction, stopRefreshAction) => {
     };
   }, [dispatch, refreshAction, stopRefreshAction]);
 };
+
+export const sizeUnits = [
+  { label: 'ki', value: 'Ki' },
```

Comment:
`ki` => `KiB`

I know that I told you that it was a small `k`, but actually it's more complicated:
- a small `k` for [kilobyte](https://en.wikipedia.org/wiki/Kilobyte) (`kB`) 
- a big `K` for [kibibyte](https://en.wikipedia.org/wiki/Kibibyte) (`KiB`)